webapp/src/utils/filterData.ts

```diff
@@ -19,4 +19,9 @@ export const getProjectList = (data: ProjectDetails[]) => {
     return projectList;
 };
 
-export const getCurrentChannelSubscriptions = (data: SubscriptionDetails[], channelID: string): SubscriptionDetails[] => data.filter(((subscription) => subscription.channelID === channelID));
+export const getCurrentChannelSubscriptions = (data: SubscriptionDetails[], channelID: string) => (data || []).filter(((subscription) => subscription.channelID === channelID));
```

Comment:
`data` is required and `typeof data is array` then why do we need `(data || []).filter...`? Can we not use `data.filter...`

Reply:
this is a change of @ayusht2810 can you look into this and try replacing this with the one which you have fixed or is more recent in any branch